servers/src/mining/stratumserver.rs

```diff
@@ -307,22 +307,30 @@ impl StratumServer {
 					let worker_stats_id = stratum_stats
 						.worker_stats
 						.iter()
-						.position(|r| r.id == workers_l[num].id)
-						.unwrap();
-					stratum_stats.worker_stats[worker_stats_id].last_seen = SystemTime::now();
+						.position(|r| r.id == workers_l[num].id);
```

Comment:
Should we just ignore such worker? I'm trying to understand how we can get into this situation, don't see it so far.

Reply:
The initial guessing has been that there is some miner (software, or class of user) that is fairly uncommon, and that manages to crash the stratum server silently. Truly crashing would allow pool operators to have the service autorestart. Or not crashing might hide the issue. So, maybe log this case loudly?